xmodule/modulestore/mixed.py

```diff
@@ -295,6 +294,11 @@ def get_course_summaries(self, **kwargs):
                     log.warning(
                         "Modulestore %s have duplicate courses %s; skipping from result.", store, course_id
                     )
+                # Skip course if it's from old mongo modulestore.
+                elif course_id.deprecated:
+                    log.warning(
+                        "Course %s from %s; skipping from result.", course_id, store
```

Comment:
Let's add more descriptive log string to mention that course is skipped due to it's oldesty :)